### DISCUSSION TITLE
Disable dark mode for tables only (v3)

### DISCUSSION THREAD

**Discussion by ItsRD** [filamentphp/filament#8337] [Help] [answered]
I'm only using tables and forms for my custom CRM and I was wondering how to disable dark mode?

Can't find any documentation on disabling it for tables.

**Comment by danharrin** [comment, marked as answer]
`darkMode: 'class'` in your Tailwind config, read the installation docs

**Reply by leandrocfe** [reply]
disable dark mode on v3?
Probably you have a class "dark" in your html

**Reply by ItsRD** [reply]
Thanks, I didn't check the tailwind config :) 

Also since there was dark mode documentation in "panel", I thought it would be a filamentphp setting

**Reply by danharrin** [reply]
> Probably you have a class "dark" in your html

Filament has dark classes in its HTML, and by default Tailwind will use them if the device is also in dark mode. `darkMode: class` prevents that behaviour

**Reply by leandrocfe** [reply]
Hey Dan,
I said this because `tailwind.config.preset` has `darkMode: 'class'`
```js
import preset from './vendor/filament/support/tailwind.config.preset'
module.exports = {
    presets: [preset],
...
```

**Reply by danharrin** [reply]
Maybe they are not using the preset
